app/views/hiring_staff/organisations/show.html.haml

```diff
@@ -2,18 +2,19 @@
 
 .school.govuk-grid-row
   .govuk-grid-column-full
-    %h1.govuk-heading-l{ class: 'govuk-!-margin-bottom-2' }
+    %h1.govuk-heading-l{ class: 'govuk-!-margin-bottom-1' }
       = t('schools.jobs.index', organisation: @organisation.name)
     - if @multiple_organisations
       - if AuthenticationFallback.enabled?
         = link_to t('sign_in.organisation.change'), auth_email_change_organisation_path, class: 'govuk-link'
       - else
         = link_to t('sign_in.organisation.change'), auth_dfe_callback_path, class: 'govuk-link'
 
-  .govuk-grid-column-full{ class: 'govuk-!-margin-top-9' }
-    .govuk-form-group.create-a-job
+  .govuk-grid-column-full{ class: 'govuk-!-margin-top-5' }
+    .govuk-form-group
       - if @organisation.vacancies.active.any?
-        = link_to t('buttons.create_job'), new_organisation_job_path, class: 'govuk-button govuk-!-margin-bottom-0 govuk-!-margin-top-0'
+        .create-a-job
```

Comment:
Rather than a div around a link, you can add the class to the link_to helper directly

Reply:
i decided to keep it as a container div as this will be the container for other buttons that will live in there to hide/show filters. i think ill rename the container to be a bit more descriptive